app/controllers/template-management/edit.js

```diff
@@ -110,10 +110,17 @@ import {
   mandateeTableView,
 } from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/mandatee-table-plugin/node';
 
+import {
+  osloLocation,
+  osloLocationView,
+} from '@lblod/ember-rdfa-editor-lblod-plugins/plugins/location-plugin/node';
+
 const SNIPPET_LISTS_IDS_DOCUMENT_ATTRIBUTE = 'data-snippet-list-ids';
 const GEMEENTE_CLASSIFICATION_URI =
   'http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/5ab0e9b8a3b2ca7c5e000001';
 
+export const OCMW =
```

Comment:
Why the `OCMW` variable?

Reply:
ah it was used in a dynamic getter to get the default municipality filter in GN, but then I realized that such a filter makes no sense in this app, but forgot to remove